libcxx/utils/libcxx/test/features.py

```diff
@@ -317,6 +317,18 @@ def _getAndroidDeviceApi(cfg):
             AddSubstitution("%{clang-tidy}", lambda cfg: _getSuitableClangTidy(cfg))
         ],
     ),
+    # Whether module support for the platform is available.
+    Feature(
+        name="has-no-module-support",
```

Comment:
```suggestion
        name="has-no-cxx-module-support",
```

To disambiguate with clang modules?